create_keepalived_sysext.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
+
+if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 VERSION SYSEXTNAME"
+  echo "The script will download the keepalived from git, checkout the tag (e.g., for v2.2.8), build a static binary and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
+  echo "A temporary directory named SYSEXTNAME in the current folder will be created and deleted again."
+  echo "All files in the sysext image will be owned by root."
+  echo "The build process requires docker"
+  "${SCRIPTFOLDER}"/bake.sh --help
+  exit 1
+fi
+
+VERSION="$1"
+SYSEXTNAME="$2"
+
+if ! which docker &>/dev/null; then
+  echo Missing docker in path
+  exit 1
+fi
+
+export ARCH=x86-64
+mkdir -p "${SYSEXTNAME}"
+cat >"${SYSEXTNAME}"/build.sh <<EOF
+apk --no-cache add \
+        binutils \
+        file \
+        file-dev \
+        gcc \
+        glib \
+        glib-dev \
+        ipset \
+        ipset-dev \
+        iptables \
+        iptables-dev \
+        libmagic-static \
+        libmnl-dev \
+        libnftnl-dev \
+        libnl3-static \
+        libnl3-dev \
+        make \
+        musl-dev \
+        net-snmp-dev \
+        openssl \
+        openssl-dev \
+        openssl-libs-static \
+        pcre2 \
+        pcre2-dev \
+        autoconf \
+        automake zlib-static  alpine-sdk linux-headers libmnl-static git
+cd /opt
+git clone https://github.com/acassen/keepalived.git
+set -ex && \
+    cd /opt/keepalived && git checkout $VERSION && \
+    ./autogen.sh && \
+    CFLAGS='-static -s' LDFLAGS=-static ./configure  --disable-dynamic-linking \
+    --prefix=/usr \
+    --exec-prefix=/usr \
+    --bindir=/usr/bin \
+    --sbindir=/usr/sbin \
+    --sysconfdir=/usr/etc \
+    --datadir=/usr/share \
+    --localstatedir=/var \
+    --mandir=/usr/share/man \
+    --enable-bfd \
+    --enable-nftables \
+    --enable-regex \
+    --enable-json  --with-init=systemd --enable-vrrp --enable-libnl-dynamic
+set -ex && \
+    cd /opt/keepalived && \
+    make && \
+    make DESTDIR=/install_root install && \
+    find /install_root && \
+    rm -rf /install_root/usr/share /install_root/usr/etc/keepalived/samples && chown $(id -u):$(id -g) /install_root -R
+EOF
+chmod +x "${SYSEXTNAME}"/build.sh
+docker run -v "${SCRIPTFOLDER}"/"${SYSEXTNAME}":/install_root/  --rm -it alpine:3.19 /bin/sh -c /install_root/build.sh
```

Comment:
```suggestion
docker run -v "${SCRIPTFOLDER}"/"${SYSEXTNAME}":/install_root/  --rm -it "${IMG}" /bin/sh -c /install_root/build.sh
```

Reply:
Should I use $(PWD)? What If the script is run from outside the current directory?

Reply:
The current directory is the one where the script is started from, not where it resides as file, and this is where `mkdir -p "${SYSEXTNAME}"` creates the folder